Update XNNPACK library to et_v20

digantdesai opened the pull request:
Summary:
From et_v18 to et_v19
* k_stride scaling fix for 4b weight
* qd8 quant param fix for memory planning

Differential Revision: D55537663

mikekgfb reviewed (approved):
Please ensure all tests pass before landing (or at least a superset of tests passing before landing, partial progress > no progress)

digantdesai committed:
Update to ExecuTorch/XNNP submodule to et_v20 (#2770)

Summary:
Pull Request resolved: https://github.com/pytorch/executorch/pull/2770

https://github.com/digantdesai/XNNPACK/releases/tag/et_v19

From et_v18 to et_v19
* k_stride scaling fix for 4b weight
* qd8 quant param fix for memory planning

From et_v19 to et_v20
* update qd8_f32_qb4w operator test harness

Reviewed By: mikekgfb

Differential Revision: D55537663
